base_geoengine_demo/views/retail_machine.xml

```diff
@@ -110,37 +101,37 @@
   <field name="begin_color">#FF680A</field>
 </record>
 <record id="geoengine_vector_layer_retailmachinestatecolored0" model="geoengine.vector.layer">
-  <field name="geo_field_id" ref="base_geoengine_demo.field_geoengine_demo_automatic_retailing_machine_the_point"/>
+  <field name="geo_field_id" ref="base_geoengine_demo.field_geoengine_demo_automatic_retailing_machine__the_point"/>
   <field name="name">Retail Machine State colored</field>
   <field name="classification">unique</field>
   <field eval="6" name="sequence"/>
   <field name="view_id" ref="ir_ui_view_retailmachineview0"/>
   <field name="geo_repr">colored</field>
   <field eval="1" name="nb_class"/>
-  <field name="attribute_field_id" ref="base_geoengine_demo.field_geoengine_demo_automatic_retailing_machine_state"/>
+  <field name="attribute_field_id" ref="base_geoengine_demo.field_geoengine_demo_automatic_retailing_machine__state"/>
```

Comment:
I didn't understand why you add a new `_` before `_state`

Reply:
This is new in Odoo 12.0, xmlids of fields, now we have 2 underscores between the name of the model and the name of the field.

Reply:
ok thanks